packages/v4/src/content/design-guidelines/content/units-and-symbols.md

```diff
@@ -0,0 +1,102 @@
+---
+id: Units and symbols
+section: UX writing
+---
+
+This page focuses on how to write units and symbols in an interface. For guidance on writing and displaying numeric values like date, time, and currency, see our [Numerics](/ux-writing/numerics) resource.
+
+Use consistent formatting, terminology, and symbols to display units of measure in your UI. Units of measure and symbols often appear when describing quantitative values including:
+
+- Date and time
+- Data storage
+- Data bandwidth
+- Currency
+
+When providing quantitative values in your UI, use the International System of Units (SI units), so that your units of measure can be understood by global users. Use the following base units and symbols to describe different quantities:
```

Comment:
```suggestion
When providing quantitative values in your UI, use the International System of Units (SI units) so that your units of measure can be understood by global users. Use the following base units and symbols to describe different quantities:
``` 
"So that" doesn't need a comma before it (unlike "so")